src/theme.ts

```diff
@@ -182,9 +173,7 @@ export default {
 			},
 			zIndex: 45,
 			// TODO: Border color should be tertiary.main once the TODOs in the Select component are resolved.
-			extend: `color: ${colors.secondary.main}; border: 1px solid ${
```

Comment:
There are a bunch of places where we use `quartenary.dark` but now it should be `quartenary.main`.

Reply:
I see thanks @sradevski 	Will update them.
Are you approaching this in a different manner? Want to merge our changes or close this PR?

Reply:
No, I just reverted the theme change and added `quartenary` to the storybooks, so you can go ahead with this and I will separately submit a PR.